### PR TITLE
DM-30938: set the timestamp field of the actuators telemetry topic, if the field is present

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/mthexapod/hexapod_csc.py [hunk 1]
```diff
@@ -213,12 +213,12 @@ def __init__(
...
-        # once ts_xml 9.1 is used everywhere.
-        self._electrical_has_motor_fields = hasattr(
+        # TODO DM-30952: remove this attribute and the code that uses it
+        # once ts_xml 9.2 is used everywhere.
+        self._actuators_has_timestamp_field = hasattr(
             self.tel_electrical.DataType(), "motorCurrent"
         )
-        if not self._electrical_has_motor_fields:
+        if not self._actuators_has_timestamp_field:
             self.log.warning(
```
Comment (hdrass) [inline python/lsst/ts/mthexapod/hexapod_csc.py:222]: It seems this warning could be updated, with the new if condition?

File: python/lsst/ts/mthexapod/hexapod_csc.py [hunk 2]
```diff
@@ -677,6 +677,7 @@ def telemetry_callback(self, server):
         server : `lsst.ts.hexrotcomm.CommandTelemetryServer`
             TCP/IP server.
         """
+        tai_unix = server.header.tai_sec + server.header.tai_nsec / 1e9
```
Comment (hdrass) [inline python/lsst/ts/mthexapod/hexapod_csc.py:680]: This is TAI time in unix format, right? Not Unix time like in here: https://en.wikipedia.org/wiki/Unix_time?

**Review by hdrass — APPROVED** [review]
Hi Russell, 
Looks very good! The changes make sense to me.
Please find my comments about some minor issues in the source code.

File: python/lsst/ts/mthexapod/hexapod_csc.py (hunk 2 above)
Comment (r-owen) [reply python/lsst/ts/mthexapod/hexapod_csc.py:680]: Yes. TAI time in unix seconds, which is the T&S standard for all timestamps.

File: python/lsst/ts/mthexapod/hexapod_csc.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/mthexapod/hexapod_csc.py:222]: Thank you for catching this. In fact I messed up the test itself (I think I accidentally reverted some code while refactoring). So I have fixed the code and the warning message.

**Pull request merged**